docs/community_resources/phpstorm_plugin.md

```diff
@@ -0,0 +1,173 @@
+# Ibexa DXP plugin for PhpStorm
+
+Ibexa DXP plugin for PhpStorm helps you to work with Ibexa DXP by speeding up installation
+and providing file templates, intentions, autocompletion, and other features.
+
+## Requirements
+
+- PhpStorm 2021.1.1 or newer
+- Enabled Symfony support plugin
+
+## Install PhpStorm plugin
+
+You can install the Ibexa DXP plugin for PhpStorm from the JetBrains Marketplace,
+or manually, from a downloaded .jar file.
+
+### Install from JetBrains Marketplace
+
+To install plugin from JetBrains marketplace:
+
+1\. In PhpStorm settings, select **Plugins** > (gear icon) > **Manage Plugin Repositories**.
```

Comment:
```suggestion
1\. In PhpStorm settings, select **Plugins** > (gear icon) > **Manage Plugin Repositories**.
```

On Mac it is "Preferences", not "Setings". So... "In PhpStorm, depending on your system platform, select **Preferences** or **Settings**."

Also, instead of pasting a screenshot that we do not have control of (it may change with the very next release of Storm, or user may be using a different theme), I'd describe the location of the gear icon - something along the lines of:
 "Select **Plugins**, then, in the top section of the window, select the gear icon, and, in the drop-down menu, click **Manage Plugin Repositories**"